app/konnect/vitals/overview-dashboard.md

```diff
@@ -2,22 +2,29 @@
 title: Overview Dashboard
 no_version: true
 content_type: reference
+badge: plus
 ---
 
-The Overview Dashboard provides metrics for Services cataloged by Service Hub within a selected time interval for the following categories: 
+The Overview Dashboard provides metrics for services cataloged by Service Hub within a selected time interval for the following categories:
 
 * **Traffic**: Total number of HTTP requests.
-* **Errors**: Percentage of failed HTTP requests. 
+* **Errors**: Percentage of failed HTTP requests.
 
 Both categories measure trends by comparing metrics across fixed comparable time intervals and plotting the data points. For example, hour-over-hour, day-over-day, week-over-week, and month-over-month.
 
 You can view a graph for each category by clicking **Traffic** or **Errors**, and switching between the two views.
 
-* **Traffic**: This graph displays the total number of HTTP requests categorized by successful and failed requests over the specified time interval. Successful requests contain all requests that returned a 1xx-3xx status code. Failed requests contain all requests that returned a 4xx-5xx status code.
+* **Traffic**: This graph displays the total number of HTTP requests categorized by successful and failed requests over the specified time interval. Successful requests contain all requests that returned a **1xx-3xx** status code. Failed requests contain all requests that returned a **4xx-5xx** status code.
```

Comment:
Should we add a sentence that explicitly mentions that 6xx codes or custom status codes are not included, or does listing the two ranges do that implicitly?

Reply:
Good question. Let's add one to cover all the bases - I can foresee someone looking specifically for info on 6xx via search.